### ISSUE TITLE
Update correlation function documentation re non-Lorentz-corrected data (Trac #1225)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#1254]
Two users have queried whether the correlation function analysis in SasView is either: a) suitable for slit-smeared data, or b) performs a Lorentz-correction?

The answer to both is no, and the documentation should be updated to reflect this to avoid future confusion.

Migrated from http://trac.sasview.org/ticket/1225
```json
{
    "status": "closed",
    "changetime": "2019-02-15T14:02:47",
    "_ts": "2019-02-15 14:02:47.462143+00:00",
    "description": "Two users have queried whether the correlation function analysis in SasView is either: a) suitable for slit-smeared data, or b) performs a Lorentz-correction?\n\nThe answer to both is no, and the documentation should be updated to reflect this to avoid future confusion.",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2019-02-11T10:34:36",
    "component": "SasView",
    "summary": "Update correlation function documentation re non-Lorentz-corrected data",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.3.0",
    "owner": "smk78",
    "type": "task"
}
```

**Comment by smk78** [comment]
Trac update at `2019/02/15 11:36:46`: 
* **smk78** commented:

In changeset 4d0666809b8cd216e5fe489e7e0b84ee88c2e188:

```text
#!CommitTicketReference repository="sasview" revision="4d0666809b8cd216e5fe489e7e0b84ee88c2e188"
Added note about Lorentz-correction to corfunc_help.rst Fixes #1254
```


* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "new" to "closed"

**Comment by smk78** [comment]
Trac update at `2019/02/15 14:02:47`: **smk78** commented:

Also updated the tutorial, but see #1256.
